Examples/ArcGISToolkitExamples/TimeSliderExample.swift

```diff
@@ -0,0 +1,96 @@
+//
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGISToolkit
+import ArcGIS
+
+class TimeSliderExample: MapViewController {
+    
+    private var map = AGSMap(basemap: AGSBasemap.topographic())
+    private var timeSlider = TimeSlider()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Set a map with ESRI topographic basemap to mapView
+        mapView.map = map
+        
+        // Configure time slider
+        timeSlider.isHidden = true
+        timeSlider.addTarget(self, action: #selector(TimeSliderExample.timeSliderValueChanged(timeSlider:)), for: .valueChanged)
+        view.addSubview(timeSlider)
+        
+        //
+        // Add constraints to position the slider
+        let margin: CGFloat = 10.0
+        timeSlider.translatesAutoresizingMaskIntoConstraints = false
+        timeSlider.bottomAnchor.constraint(equalTo: mapView.attributionTopAnchor, constant: -margin).isActive = true
+        
+        if #available(iOS 11.0, *) {
+            let layoutMarginsGuide = view.layoutMarginsGuide
+            timeSlider.leadingAnchor.constraint(equalTo: layoutMarginsGuide.leadingAnchor).isActive = true
```

Comment:
If we want to maintain the margin constant we had before, I would suggest using the view's `safeAreaLayoutGuide` over the view's `layoutMarginsGuide`. The layout margins guide is Apple's recommend content layout margin and as far as i'm aware we cannot change the constant value whereas the safe area layout guide is the layout guide that determines where in the screen is safe to interact with and thus will allow us to reapply our desired 10.0 pt margin.
```
timeSlider.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: margin).isActive = true
timeSlider.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -margin).isActive = true
```

Reply:
fixed